corporate/lib/stripe.py

```diff
@@ -3139,11 +3143,8 @@ def is_sponsored(self) -> bool:
 
     @override
     def get_metadata_for_stripe_update_card(self) -> Dict[str, Any]:
-        return {
-            "type": "card_update",
-            # TODO: Add user identity metadata from the remote realm identity
-            # "user_id": user.id,
-        }
+        assert self.remote_billing_user is not None
+        return {"type": "card_update", "user_id": str(self.remote_billing_user.id)}
```

Comment:
Hmm, I don't like having a bare `user_id` here as that is the same type for `RealmBillingSessions`, and on the Stripe side it won't be obvious what customer type it is; are there restrictions on what fields we can add here `remote_realm_user_id` as the variable name.